projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -7,6 +7,7 @@ import { useSelect } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { isComingSoon, isPrivateSite } from '@automattic/jetpack-shared-extension-utils';
+import { numberFormat } from '@automattic/jetpack-components';
```

Comment:
I think we should look at a more compact display at some point, like https://github.com/Automattic/wp-calypso/tree/9fb7ac149e55361c0c3a9639ad0caa5e15fc88b0/client/lib/format-number-compact

Reply:
Yeah, that would be nice to implement something like this in our `numberFormat`. I've created an issue to keep track of this:
https://github.com/Automattic/jetpack/issues/24554

We purposefully got rid of the `i18n-calypso` dependency in Jetpack a while ago, so we would be implementing this a bit differently.